common/djangoapps/track/tests/test_middleware.py

```diff
@@ -60,6 +60,7 @@ def test_default_request_context(self):
             'session': '',
             'username': '',
             'ip': '127.0.0.1',
+            'remote_addr': '127.0.0.1',
```

Comment:
127.0.0.1 seems like a funny choice for a test case, but my first test case would have something more generic.